### PR TITLE
rebar3: 3.15.1 -> 3.16.0

### PR DESCRIPTION
**Pull request by dlesl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123906` at 424212d2 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>lfe_1_2</li>
  </ul>
</details>
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li>elixir_ls</li>
    <li>erlang-ls</li>
    <li>lfe (lfe_1_3)</li>
    <li>rebar3</li>
    <li>relxExe</li>
  </ul>
</details>

File: pkgs/development/tools/build-managers/rebar3/default.nix [hunk 1]
```diff
@@ -16,19 +16,12 @@ let
...
 
-    patches = [
-      # TODO: remove this on next rebar3 release
-      (fetchpatch {
-        name = "escriptize-erl-libs";
-        url = "https://github.com/erlang/rebar3/commit/8049a92512420b0967a4c23acfa304d8ca7a712e.patch";
-        sha256 = "0jzdy7n2nz4v38nn76bgjcmssvqgvdhy9v2gl867ylwqn1y5sdm1";
-      })
-    ];
+    patches = [ ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/rebar3/default.nix:25]:
```suggestion
```

File: pkgs/development/tools/build-managers/rebar3/default.nix [hunk 2]
```diff
@@ -16,19 +16,12 @@ let
...
       rev = version;
-      sha256 = "1pcy5m79g0l9l3d8lkbx6cq1w87z1g3sa6wwvgbgraj2v3wkyy5g";
+      sha256 = "1yqvm37l5rn5dyg4sc2hv47930s2524qrdpnjwy3zqa27r7k5n36";
     };
 
     buildInputs = [ erlang ];
 
-    patches = [
-      # TODO: remove this on next rebar3 release
-      (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/rebar3/default.nix:26]: Please remove the unused fetchpatch from inputs.

**Comment by happysalada** [comment]
Result of `nixpkgs-review pr 123906` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>lfe_1_2</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>elixir_ls</li>
    <li>erlang-ls</li>
    <li>lfe (lfe_1_3)</li>
    <li>rebar3</li>
    <li>relxExe</li>
  </ul>
</details>

**Pull request merged**